### PR TITLE
Add sbscription statuses filter

### PR DESCRIPTION
**Pull request by phieronymus**
Linked to https://univapay-systems.slack.com/archives/CA0U3CSQ1/p1703159104669109

### CODE REVIEW

File: src/resources/Subscriptions.ts [hunk 1]
```diff
@@ -78,6 +78,7 @@ export type SubscriptionPlanItem =
 export type SubscriptionsListParams = CRUDPaginationParams & {
     search?: string;
     status?: SubscriptionStatus;
```
Comment (gpapako) [inline src/resources/Subscriptions.ts:80]: Do we still keep the `status` ? Or are we gonna switch to using `statuses` only from now on?

**Review by gpapako — COMMENTED** [review]
Just a question and then we can merge

File: src/resources/Subscriptions.ts (hunk 1 above)
Comment (phieronymus) [reply src/resources/Subscriptions.ts:80]: API did not remove it so we have no reason to not support it any longer if a customer wants to use it.

Comment (gpapako) [reply src/resources/Subscriptions.ts:80]: yes, of course. Was just checking if the API removed it or not. 👍

**Review by gpapako — APPROVED** [review]
👍

**Pull request merged**